index.bs

```diff
@@ -9589,6 +9619,76 @@ The [=remote end steps=] given |session|, and |command parameters| are:
 
 </div>
 
+### Events ### {#module-input-events}
+
+#### The input.fileDialogOpened Event #### {#event-input-fileDialogOpened}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl remote-cddl">
+         input.FileDialogOpened = (
+            method: "input.fileDialogOpened",
+            params: input.FileDialogInfo
+         )
+
+         input.FileDialogInfo = {
```

Comment:
Should we provide an info if the file dialog was intercepted or not? And if so, should the value be different for different sessions depending on the `intercept file dialogs` capability?

Reply:
The dialog is a shared resource so it will be intercepted if any of the active session intercepts it. I think a flag would be useful but there is no immediate use case for it.